code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -110,6 +110,21 @@
 	eat()
 	return
 
+/obj/singularity/wizard/attack_tk(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		GET_COMPONENT_FROM(insaneinthemembrane, /datum/component/mood, C)
+		if(insaneinthemembrane.sanity < 15)
+			return //they've already seen it and are about to die, or are just too insane to care
+		to_chat(C, "<span class='userdanger'>OH GOD! NONE OF IT IS REAL! NONE OF IT IS REEEEEEEEEEEEEEEEEEEEEEEEAL!</span>")
+		insaneinthemembrane.sanity = 0
+		for(var/lore in typesof(/datum/brain_trauma/severe))
+			C.gain_trauma(lore)
+		sleep(100)

```

Comment:
you need to make sure they exist after the sleep

Reply:
expect a fix very soon!

Reply:
i'd advise moving this into a timer too

Reply:
that seems a lot more sane, give me a sec.